enhancements/sparse-manifest-list.md

```diff
@@ -0,0 +1,111 @@
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions [optional]
+
+> 1. What are the potential impacts on Quay's existing storage backend when implementing sparse manifest lists?
+> 2. How will this enhancement affect existing Quay users, and what migration steps are necessary?
+> 3. What changes will be required in client software to be compatible with handling sparse manifest lists? This includes Podman and OC Mirror
+
+## Summary
+
+The introduction of sparse manifest list support in Quay container registry aims to increase the efficiency of how container images are stored and managed. This enhancement focuses on optimizing the use of storage space and minimizing the data transfer when pulling images from the registry. The feature will be particularly beneficial for users with large image repositories or those operating in bandwidth-constrained environments.
+
+## Motivation
+
+With the growing size of container images and the increasing need for efficiency in container registry operations, there's a clear need for more advanced storage solutions. Sparse manifest lists offer a compelling approach to address these challenges. This enhancement seeks to leverage these benefits to improve Quay's functionality and user experience. This will also help reduce costs on Openshift in supporting OCP installations in various architectures.
+
+### Goals
+
+- Implement sparse manifest list support in Quay container registry.
+- Optimize storage utilization and data transfer efficiency.
+- Ensure backward compatibility and minimal disruption for current users.
+- Ensure that existing clients are able to handle pulling these images, and that other clients that do not support this feature necessarily are not broken
+
+### Non-Goals
+
+- Overhaul of Quay's existing storage architecture.
+- Disruption of existing workflows for current Quay users.
+
+## Proposal
+
+The proposal includes developing and integrating sparse manifest list support into the Quay container registry. This will involve changes in how Quay stores and handles container images, focusing on layer deduplication and efficient data management.
+
+### User Stories [optional]
+
+#### Story 1
+
+A user with a large repository of container images can significantly reduce their storage footprint by using sparse manifest lists, as common layers across different images are stored only once.
```

Comment:
(I know ~noting about Quay)

I can’t see how sparse manifest lists change anything about this. Removing a per-platform image instance can never add a new layer sharing opportunity.

Is this saying that, _independently_ from accepting sparse manifests list, the scope of layer sharing is going to be increased?